src/components/shared/Button.jsx

```diff
@@ -0,0 +1,72 @@
+import styled, { css } from "styled-components";
+
+const BUTTON_SIZES = {
+  md: css`
+    --button-min-width: 240px;
+    --button-min-height: 64px;
+    --button-font-size: 1rem;
+    --button-padding: 12px 24px;
+    --button-border: 2px solid #000;
+    --button-radius: 4px;
+  `,
+  lg: css`
+    --button-min-width: 420px;
+    --button-min-height: 64px;
+    --button-font-size: 1rem;
+    --button-padding: 12px 24px;
+    --button-border: 4px solid #000;
```

Comment:
P1: 목업 화면과 동일하게 border 픽셀을 맞춰주시면 좋을 것 같습니다.
추가적으로 해당 버튼 내의 볼드체로 변경 부탁드립니다.

```suggestion
    --button-border: 2px solid #000;
```

Reply:
확인했습니다!
수정 하겠습니다🥲

Reply:
미처 발견하지 못했던 디테일한 부분 지적 좋습니다 👍